Gruntfile.js

```diff
@@ -57,6 +57,12 @@ module.exports = function(grunt) {
                     prefix: '@version *'
                 },
                 src: ['dist/plugins/soundjs/*.js']
+            },
+            highlander: {
+                options: {
+                    prefix: '@version *'
```

Comment:
I feel pretty strongly that we should start versioning the plugins independently. We probably should've when we added a second one, but there's no accounting for Tennessee.